modules/configuring-a-pipeline-server.adoc

```diff
@@ -68,6 +68,8 @@ The *Configure pipeline server* dialog appears.
 [IMPORTANT]
 ====
 If you specify incorrect data connection settings, you cannot update these settings on the same pipeline server. Therefore, you must delete the pipeline server and configure another one.
+
+If you plan to import artifacts that were not generated by tasks in your pipeline, or you plan to use external artifacts that were not generated by any pipeline, you can only import these artifacts to the S3-compatible object storage bucket that you define in the *Bucket* field.
```

Comment:
If you wish to use an existing artifact that was not generated by a task in the current pipeline you can use a [dsl.importer](https://kubeflow-pipelines.readthedocs.io/en/latest/source/dsl.html#kfp.dsl.importer) component to load the artifact from its URI.

Reply:
I can update this to mention the dsl importer component. However, we also need to mention that you can only import these artifacts to the S3-compatible object storage bucket that you define in the *Bucket* field in the pipeline configuration, as that was the point of the documentation request.

Reply:
I see

```
If you wish to use an existing artifact that was not generated by a task in the current pipeline you can use a [dsl.importer](https://kubeflow-pipelines.readthedocs.io/en/latest/source/dsl.html#kfp.dsl.importer) component to load the artifact from its URI. You can only import these artifacts to the S3-compatible object storage bucket that you define in the *Bucket* field.
```